packages/venia-ui/lib/util/images.js

```diff
@@ -57,17 +57,18 @@ export const generateUrlFromContainerWidth = (
     );
 };
 
-export const generateSrcset = (imageURL, type) => {
+export const generateSrcset = (imageURL, type, ratio) => {
     if (!imageURL || !type) return '';
 
+    const imageRatio = ratio || DEFAULT_WIDTH_TO_HEIGHT_RATIO;
```

Comment:
This API would need to change, so maybe drilling ratio is still best to keep things simple.

Reply:
I have made sure to add default props for the ratio in the components that end up using this util.